File: Núcleo/src/LogicaNegocio/IRecepcionista.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package LogicaNegocio;

import java.util.Date;
import java.util.List;

/**
 *
 * @author mariolpz
 */
public interface IRecepcionista {
    
    public boolean registrar(Recepcionista recepcionista);
    public boolean modificar(Recepcionista recepcionista);
    public boolean registrarEntrada(String numeroConsultorio, String numeroPersonal);
    public boolean registrarSalida(String numeroPersonal);
    public boolean eliminar(String numeroPersonal);
    public boolean agregarConsulta(Consulta consulta);
    public List obtenerCitas(Date fecha);
    public Recepcionista obtenerRecepcionista(String numeroPersonal);
}

File: Nucleo servidor web/application/libraries/IRecepcionista.php
<?php
interface IRecepcionista{
    public function registrar($recepcionista, $idUsuario);
    public function modificar($recepcionista);
    public function registrarEntrada($numeroConsultorio, $rfc);
    public function registrarSalida($rfc);
    public function eliminar($rfc);
    public function agregarConsulta($consulta);
    public function obtenerCitas($fecha);
    public function obtenerRecepcionista($rfc);
    public function obtenerRecepcionistaId($idUsuario);
}
